tools/mksurfdata_map/src/mkpctPftTypeMod.F90

```diff
@@ -511,6 +510,49 @@ end subroutine check_vals
   ! Module-level routines (not member functions)
   ! ========================================================================
 
+  !-----------------------------------------------------------------------
+  subroutine set_max_p2l_array(pct_pft_max_arr,pct_pft_arr)
+    !
+    ! !DESCRIPTION:
+    ! Given an array of pct_pft_type variables, set all max_p2l.
```

Comment:
As with the comment on the subroutine name, the description should make it clear that it's (possibly) updating existing max values.